controller/controller.go

```diff
@@ -121,6 +121,8 @@ func respondWithError(w http.ResponseWriter, err error) {
 }
 
 func appHandler(c handlerConfig) http.Handler {
+	loadSchemas()
```

Comment:
What do you think about renaming the package to `schema` or `schemas`? Then we can do `schema.Load()`